.gitlab-ci.yml

```diff
@@ -162,7 +162,8 @@ build:master_docker_latest:
     - build:dev_docker_latest
     - build:master_docker_latest
   script:
-    - ./test/toolbox_image.sh ".build_"*"/um-update"*".xz.img"
+    - ./test/toolbox_image.sh .build_*/um-update*.xz.img
+    - ./test/prepare_disk.sh .build_*/um-update*.xz.img
```

Comment:
I think this was an inintended change from merging, also also quote the new addition :)

Reply:
This is not intended.

Reply:
as this was also fixed in EMP-406, marking as resolved.